libs/dh/message-archive/feature-search/src/lib/start.component.ts

```diff
@@ -119,6 +120,7 @@ export class DhMessageArchiveSearchStartComponent {
   form = inject(DhMessageArchiveSearchFormService);
   searchChanged = output<GetArchivedMessagesQueryVariables>();
   modal = viewChild.required(WattModalComponent);
+  autoOpen = input(false);
```

Comment:
question: does it have to be an input? you can just do this if it should always "auto open":

```html
<watt-modal autoOpen />
```

Reply:
Should be simplified now :)